makefiles/pseudomodules.inc.mk

```diff
@@ -479,6 +479,7 @@ PSEUDOMODULES += stdio_telnet
 PSEUDOMODULES += stdio_uart_onlcr
 ## @}
 PSEUDOMODULES += stdio_uart_rx
+PSEUDOMODULES += stdio_usb_serial_jtag_rx
```

Comment:
Since this STDIO backend is only working for ESP32x, it should be defined in `cpu/esp32/Makefile.include`. The dependencies for this module are also defined in `cpu/esp32/Makefile.dep`.